src/yb/client/tablet_server.h

```diff
@@ -84,12 +85,18 @@ struct YBTabletServerPlacementInfo {
 };
 
 struct YBActiveUniverseHistoryInfo {
```

Comment:
Looks very Similar to wait_state.h/WaitStateInfo.

Why do we need this separately here?